modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/ProcessResourceTest.java

```diff
@@ -74,6 +77,32 @@ public void testGetProcess() throws Exception {
 		super.testGetProcess();
 	}
 
+	@Override
+	@Test
+	public void testGetProcessLastCheckedDate() throws Exception {
+		Process process = testGetProcess_addProcess();
+
+		Instance instance = _workflowMetricsRESTTestHelper.addInstance(
+			testGroup.getCompanyId(), process.getId());
+
+		String mostRecentDate = "20210426142015";
+
```

Comment:
@MarinhoFeliphe, Use a Date object instead of String and call it just `lastSLACheckDate`. Thx